virtualcluster/pkg/syncer/resources/pod/dws.go

```diff
@@ -55,15 +55,22 @@ func (c *controller) Reconcile(request reconciler.Request) (res reconciler.Resul
 	reconcilestart := time.Now()
 	targetNamespace := conversion.ToSuperClusterNamespace(request.ClusterName, request.Namespace)
 
-	pPod, err := c.podLister.Pods(targetNamespace).Get(request.Name)
-	if err != nil && !apierrors.IsNotFound(err) {
-		return reconciler.Result{Requeue: true}, err
-	}
-
 	vPod := &corev1.Pod{}
 	if err := c.MultiClusterController.Get(request.ClusterName, request.Namespace, request.Name, vPod); err != nil && !apierrors.IsNotFound(err) {
 		return reconciler.Result{Requeue: true}, err
 	}
+	if featuregate.DefaultFeatureGate.Enabled(featuregate.TenantScheduler) {
+		// if constants.LabelTenantIgnoreSync is true, bypass syncing
+		schedulerlabel, ok := vPod.GetLabels()[constants.LabelTenantIgnoreSync]
```

Comment:
can you rename the variable name "schedulerlabel"? we should not imply scheduling anywhere.

Reply:
Sure